core/toolbox/toolbox.js

```diff
@@ -346,6 +346,202 @@ Blockly.Toolbox.prototype.addToolboxItem_ = function(toolboxItem) {
   }
 };
 
+/**
+ * Adds a style on the toolbox. Usually used to change the cursor.
+ * @param {string} style The name of the class to add.
+ * @protected
+ */
+Blockly.Toolbox.prototype.addStyle = function(style) {
```

Comment:
In the interest of simplicity, can we leave these two out. I don't see it adding any additional value to the interface since HtmlDiv is public, if we have a need to add styles to it, this can be done where-ever this is called. WDYT?

Reply:
My main concern is that if we pull these out then developers lose the ability to easily change this functionality. I personally don't think it is worth the small cleanup, but let me know if I am missing something.